.azure-pipelines/templates/variables.yml

```diff
@@ -2,5 +2,5 @@ variables:
   ubuntu_pool: 'pool-ubuntu-2204'
   ubuntu_multi_core_pool: 'pool-ubuntu-latest-multi-core'
   windows_pool: 'pool-windows-2019'
-  ubuntu_arm64_pool: 'ubuntu-arm64-2004-pool'
+  ubuntu_arm64_pool: 'pool-ubuntu-latest-arm64'
```

Comment:
I remember we decided not to use the name `latest`?

Reply:
Using the latest version saves effort by avoiding changes to the pool reference in CI and release pipelines, but it loses the ability to revert back if we encounter some issue.

How about this, we create a temp pool with newer image when we need to update the agent, then test it in the main repository. If everything works fine, we'll update the latest pool with the newer image. This could eliminate the issue we met during the 20.04 deprecation.

@wangzelin007 for awareness